tensorflow/python/ops/image_ops_impl.py

```diff
@@ -792,6 +792,14 @@ def pad_to_bounding_box(image, offset_height, offset_width, target_height,
     ValueError: If the shape of `image` is incompatible with the `offset_*` or
       `target_*` arguments, or either `offset_height` or `offset_width` is
       negative.
+      
+  Usage example:
+  ```python
+    
+    >>> x = tf.random.uniform((28, 28, 1))
+    >>> tf.image.pad_to_bounding_box(x, 28, 28, 100, 100)
+    ```
```

Comment:
Assert something about the result to make this a doctest? Like print its shape at least?

Reply:
Ok!

Reply:
done!😊

Reply:
Then put the resulting shape in the comment so this runs as a doctest

Reply:
done! If there are any other changes needed please say so. I can't wait to see this merged😊